usagov/vote-gov-drupal
# VOTE-2356: NVRF Module

## Jira ticket

[VOTE-2359](https://cm-jira.usa.gov/browse/VOTE-2359)

## Description

Creates a custom block plugin module with the NVRF React App embedded

## Deployment and testing

### QA/Testing instructions

1. Go to Extend and enable NVRF App
2. Go to block layout, select NVRF Digital Form in the list
3. Uncheck show title
4. Navigate Home to test the app
5. All functionality should be as expected, some styling inheritance issues will be fixed next sprint.

## Checklist for the Developer




## Checklist for the Peer Reviewers
